<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlopeDialogForm</class>
 <widget class="QDialog" name="SlopeDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slope</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="m_slopeTitleLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Allow to compute slope from DTM or TIN.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_inputGroupBox">
     <property name="title">
      <string>Input Layer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="m_layerLabel">
        <property name="text">
         <string>Layer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_layersComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="m_dummycheckBox">
          <property name="text">
           <string>Dummy Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_dummylineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_computegroupBox">
     <property name="title">
      <string>Compute</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_sloperadioButton">
        <property name="text">
         <string>Slope</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_aspectradioButton">
        <property name="text">
         <string>Aspect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_unitgroupBox">
     <property name="title">
      <string>Slope Unit</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="m_degreesradioButton">
        <property name="text">
         <string>Degrees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_percentradioButton">
        <property name="text">
         <string>Percent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_outputGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="m_rasterGroupBox">
        <property name="title">
         <string>Raster Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="m_resXLabel">
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="text">
            <string>Resolution: X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_resXLineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="m_resYLabel">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="m_resYLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="m_dimCLabel">
           <property name="text">
            <string>Dimension: C</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_dimCLineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="m_dimLLabel">
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="m_dimLLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QToolButton" name="m_srsToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SRS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_resSRIDLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_repositoryLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="m_repositoryLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_targetFileToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_targetDatasourceToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>BD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_layerNameLabel">
        <property name="text">
         <string>Layer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_newLayerNameLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_okPushButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="m_cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
